modules/recipes_magazin_contenta/default_content/consumers.csv

```diff
@@ -1,2 +1,7 @@
 label,uuid,owner,description,image,image_styles
-Umami Site in Elm,0eecd0b9-b149-418f-b5ef-266b9aeeddab,Contenta team,The Umami Site consumer written in Elm.,elm.png,"medium,large"
+Umami Site Elm,0eecd0b9-b149-418f-b5ef-266b9aeeddab,Contenta team,The Umami Site consumer written in Elm.,elm.png,"recipe_list_350_300,recipe_detail_750_550"
+Umami Site React,8be46df2-4cf7-48f8-ba70-6280ea62a0be,Contenta team,The Umami Site consumer written in React.,reactjs.png,"recipe_list_350_300,recipe_detail_750_550"
```

Comment:
I hope we don't cause any licensing issues by committing these images?

Reply:
I'm pretty sure that usage is OK. We're not profiting and we're mentioning the frameworks (even promoting them).